tests/inputs/empty-for-test.txt

```diff
@@ -0,0 +1 @@
+This file is empty for test purposes.
```

Comment:
@sage-wright still needed?

Reply:
Yes! Keep this file.

Reply:
This is the file that we're passing in as input for the poppunk task so it doesn't try to find the `gs://` files